index.js

```diff
@@ -1,100 +1,105 @@
-import './src/shapes/circle.class.js';
-import './src/shapes/triangle.class.js';
-import './src/shapes/ellipse.class.js';
-import './src/shapes/rect.class.js';
-import './src/shapes/polyline.class.js';
-import './src/shapes/polygon.class.js';
-import './src/shapes/path.class.js';
-import './src/shapes/group.class.js';
-import './src/shapes/active_selection.class.js'; // optional interaction
-import './src/shapes/image.class.js';
-import './src/mixins/object_straightening.mixin.js'; // optional objectstraightening
-import './src/filters/webgl_backend.class.js'; // optional image_filters
-import './src/filters/2d_backend.class.js'; // optional image_filters
-import './src/filters/base_filter.class.js'; // optional image_filters
-import './src/filters/colormatrix_filter.class.js'; // optional image_filters
-import './src/filters/brightness_filter.class.js'; // optional image_filters
-import './src/filters/convolute_filter.class.js'; // optional image_filters
-import './src/filters/grayscale_filter.class.js'; // optional image_filters
-import './src/filters/invert_filter.class.js'; // optional image_filters
-import './src/filters/noise_filter.class.js'; // optional image_filters
-import './src/filters/pixelate_filter.class.js'; // optional image_filters
-import './src/filters/removecolor_filter.class.js'; // optional image_filters
-import './src/filters/filter_generator.js'; // optional image_filters
-import './src/filters/blendcolor_filter.class.js'; // optional image_filters
-import './src/filters/blendimage_filter.class.js'; // optional image_filters
-import './src/filters/resize_filter.class.js'; // optional image_filters
-import './src/filters/contrast_filter.class.js'; // optional image_filters
-import './src/filters/saturate_filter.class.js'; // optional image_filters
-import './src/filters/vibrance_filter.class.js'; // optional image_filters
-import './src/filters/blur_filter.class.js'; // optional image_filters
-import './src/filters/gamma_filter.class.js'; // optional image_filters
-import './src/filters/composed_filter.class.js'; // optional image_filters
-import './src/filters/hue_rotation.class.js'; // optional image_filters
-import './src/shapes/text.class.js'; // optional text
-import './src/mixins/text_style.mixin.js'; // optional text
-import './src/shapes/itext.class.js'; // optional itext
-import './src/mixins/itext_behavior.mixin.js'; // optional itext
-import './src/mixins/itext_click_behavior.mixin.js'; // optional itext
-import './src/mixins/itext_key_behavior.mixin.js'; // optional itext
-import './src/mixins/itext.svg_export.js'; // optional itext
-import './src/shapes/textbox.class.js'; // optional textbox
-import './src/mixins/default_controls.js'; // optional interaction
+import { fabric } from './HEADER';
+// import './lib/event'), // optional gestures
+import './src/mixins/collection.mixin';
+
+// import './src/mixins/Observable';
+import { Observable } from './src/mixins/Observable';
+fabric.Observable = Observable;
```

Comment:
i think is safe to bring all of those up, Observable does not seem to need to come after collection.mixin.
I would also rather do the assign to fabric inside the file itself.

Reply:
> I would also rather do the assign to fabric inside the file itself.

I am against for a few reasons:
- after we complete the migration we won't be assigning to fabric
- limiting the mess to the index which will change a lot anyways